plotsScript.py

```diff
@@ -2218,36 +2233,30 @@ def cosmosWideSvmScores(trainXd=False, trainSvm=False, fontSize=18):
                                      xlabel=r'{0} HSC-I Wide {1}'.format(_strMag, name), linestyle='--',
                                      legendLabel=r'SVM', standardized=False, magRange=(18.5, 25.0), svm=True)
         dirHome = os.path.expanduser('~')
-        figScores.savefig(os.path.join(dirHome, 'Desktop/cosmosWideSvmScores{0}.png').format(name), dpi=120, bbox_inches='tight')
+        figScores.savefig(os.path.join(dirHome, 'Desktop/cosmosWideSvmScores{0}.png').format(name),
+                          dpi=120, bbox_inches='tight')
 
 
-def makeRachelPlots(depth='udeep', fontSize=18):
-    with open('clfsColsExt.pkl', 'rb') as f:
+def makeRachelPlots(depth=depth, fontSize=18):
+    with open(classifierFile, 'rb') as f:
         clfs = pickle.load(f)
     magBins = [(18.0, 22.0), (22.0, 24.0), (24.0, 25.0), (25.0, 26.0)]
     clfXd = dGauss.XDClfs(clfs=clfs, magBins=magBins)
+    cat = afwTable.SimpleCatalog.readFits(matchedCatFile)
     if depth == 'udeep':
         suptitle = 'Ultra Deep'
-        try:
-            cat = afwTable.SimpleCatalog.readFits('/scr/depot0/garmilla/HSC/udeepHscClass.fits')
-        except LsstCppException:
-            cat = afwTable.SimpleCatalog.readFits('/home/jose/Data/udeepHscClass.fits')
-    elif depth == 'wide':
-        suptitle = 'Wide'
-        try:
-            cat = afwTable.SimpleCatalog.readFits('/scr/depot0/garmilla/HSC/wideHscClass.fits')
-        except LsstCppException:
-            cat = afwTable.SimpleCatalog.readFits('/home/jose/Data/wideHscClass.fits')
+    elif depth == 'deep':
+        suptitle = 'Deep'
     else:
-        raise ValueError('I only have udeep and wide depths.')
+        raise ValueError('I only have udeep and deep depths.')
```

Comment:
The code logic here is very simplistic and needs significant revision. For now, I am simply removing the check completely and just using the 'depth' value as the suptitle of the plot.